resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
```

Comment:
@shrug @underscorgan I think we've agreed that this file path should be ok, but there's one thing that is a little wonky here right now, which is that we also need to put this path into the `global.conf` file in order for everything to work.  That file is managed in the individual clojure projects, not in ezbake, so if we do it this way we're sorting of relying on the devs to use this exact path in their packaging.  Another option might be for us to make this path a config parameter on ezbake itself, and then the devs would still have to specify the value in two places but both of those places would be in their own repos.  Any thoughts?